QSB/TransformSync/SectorSync.cs

```diff
@@ -64,17 +64,21 @@ private Sector FindSectorByName(Sector.Name sectorName, string goName)
 
         private void OnClientReceiveMessage(SectorMessage message)
         {
-            DebugLog.ToScreen($"Received sector {message.SectorName} for id {message.SenderId}");
-
             var sector = FindSectorByName((Sector.Name)message.SectorId, message.SectorName);
 
+            if (_allSectors == null || _allSectors.Count == 0)
+            {
+                DebugLog.ToConsole($"Error: _allSectors is null or empty for player {message.SenderId}!", OWML.Common.MessageType.Error);
```

Comment:
empty for local, not sender

Reply:
also, this isn't really the best place for for this check... maybe inside `if (sector == null)` below?